Makefile.deps

```diff
@@ -0,0 +1,25 @@
+YARNFLAGS :=
```

Comment:
Why are we moving this to a separate Makefile?

Reply:
Don't want to pollute "global" Make namespace with target-specific flags. Second reason: assigning variables *within* a make target is impossible / not implemented.

Turning it into a shell-script

Reply:
changed